pythonosc/dispatcher.py

```diff
@@ -54,6 +54,19 @@ def map(self, address, handler, *args, needs_reply_address=False):
     # regarding multiple mappings
     self._map[address].append(Handler(handler, list(args), needs_reply_address))
 
+  def unmap(self, address, handler, *args, needs_reply_address=False):
+    """Remove an already mapped handler from an address
+
+    Args:
+      - address: An explicit endpoint.
+      - handler: A function that will be run when the address matches with
+                 the OscMessage passed as parameter.
+      - args: Any additional arguments that will be always passed to the
+              handlers after the osc messages arguments if any.
+      - needs_reply_address: True if the handler function needs the
+              originating client address passed (as the first argument)."""
+    self._map[address].remove(Handler(handler, list(args), needs_reply_address))
```

Comment:
check for presence to avoid a KeyError exception being thrown
```
if not address in self._map:
  return
```

Reply:
I also thought about this when writing the method, but I think it's more verbose to have the code throw the exception. Either you keep track of your mapped handlers, or you can handle the exception. If it were to just return without any message, you would get no feedback that your unmapping failed/did nothing.